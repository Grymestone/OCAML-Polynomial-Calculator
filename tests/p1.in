2a * 2a^2 * 2a^2
