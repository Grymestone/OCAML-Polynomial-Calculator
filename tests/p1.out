(((2*a)*(2*(a)^2))*(2*(a)^2))
Parsed Times 
Parsed Times 
Parsed Term 
Parsed Term 
Parsed Times 
Parsed Times 
Parsed Term 
Parsed Term 
Parsed Times 
Parsed Term 
Parsed Term 
Entering Times 
Times inside Times 

 Distributing... Starting with Plus operators
(2x^0) (1x^2) *+ 
 Distributing multiplication 
(2x^0) (1x^1) *(2x^0) (1x^2) **
 Result 
(2x^0) (1x^1) *(2x^0) (1x^2) *(2x^0) (1x^2) **+Entering Times 
Times inside Times 

 Distributing... Starting with Plus operators
(2x^0) (1x^2) *(2x^0) (1x^2) *+ 
 Distributing multiplication 
(2x^0) (1x^1) *
 Result 
(2x^0) (1x^1) (2x^0) (1x^2) **(2x^0) (1x^1) (2x^0) (1x^2) **+Entering Times 
Multiplying Terms 
Result: 2x^1
Entering Times 
Multiplying Terms 
Result: 2x^1
(2x^0) (1x^2) *(2x^1) *(2x^0) (1x^2) *(2x^1) ***Degree of expression: 0 
Entering Times 
Times inside Times 

 Distributing... Starting with Plus operators
+ 
 Distributing multiplication 
(2x^0) (1x^2) *(2x^1) *(2x^0) (1x^2) *(2x^1) **
 Result 
+*Degree of expression: 0 
Entering Times 
